src/press/tests/test_views.py

```diff
@@ -31,3 +38,50 @@ def test_press_nav_account_links_do_not_have_return(self):
         content = response.content.decode()
         self.assertNotIn('/login/?next=', content)
         self.assertNotIn('/register/step/1/?next=', content)
+
+    @override_settings(URL_CONFIG='domain')
+    def test_link_from_journal_users_to_all_users_domain(self):
+        self.client.force_login(self.press_manager)
+        self.journal_one.domain = 'one.example.org'
+        self.journal_one.save()
+        response = self.client.get(
+            '/user/all/',
+            SERVER_NAME=self.journal_one.domain,
+        )
+        content = response.content.decode()
+        self.assertIn(f'{self.press.domain}/user/all/', content)
+
+    @override_settings(URL_CONFIG='path')
+    def test_link_from_journal_users_to_all_users_path(self):
+        self.client.force_login(self.press_manager)
+        response = self.client.get(
+            '/user/all/',
+            SERVER_NAME=self.journal_one.domain,
+        )
+        content = response.content.decode()
+        self.assertIn(f'{self.press.domain}/user/all/', content)
+
+    @override_settings(URL_CONFIG='domain')
+    def test_link_from_all_users_to_journal_users_domain(self):
+        self.client.force_login(self.press_manager)
+        self.journal_one.domain = 'one.example.org'
+        self.journal_one.save()
+        response = self.client.get(
+            '/user/all/',
+            SERVER_NAME=self.press.domain,
+        )
+        content = response.content.decode()
+        self.assertIn(f'one.example.org/user/all/', content)
+
+    @override_settings(URL_CONFIG='path')
+    def test_link_from_all_users_to_journal_users_path(self):
+        self.client.force_login(self.press_manager)
+        response = self.client.get(
+            '/user/all/',
+            SERVER_NAME=self.press.domain,
+        )
+        content = response.content.decode()
+        self.assertIn(
+            f'{self.press.domain}/{self.journal_one.code}/user/all/',
+            content
+        )
```

Comment:
While I appreciate the existance of tests, these types of tests we are introducing here look more like acceptance tests than unit tests. They are relatively slow and they are just testing that you've put the correct URL on the template, so I don't think we should include them as long as we use the correct template tag to generate urls and that the tag itself is properly unittested.